### PR TITLE
Create Greninja.ms

### PR DESCRIPTION
**Pull request by ThisChicken**
to prove greninja's speed: https://66.media.tumblr.com/42a17ff206b6f0be9060ffe5987b3c7a/tumblr_o72fhu8xsU1sr6y44o1_500.gif

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                "/haste 0-3",                
+                "/invis",
+                "/vis"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 11 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 2 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 21 9999 19 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: requires proof of durability

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                "/run 0-7",
+                "/haste 0-3",                
+                "/invis",
+                "/vis"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 11 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 2 true",
+                "/effect ". @plr ." 13 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: requires proof he can breathe in space/underwater

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                "/buff",
+                "/jump 0-6",
+                "/run 0-7",
+                "/haste 0-3",                
+                "/invis",
+                "/vis"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 11 true",
+                "/effect ". @plr ." 10 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: requires proof of regeneration

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            'powers': array(
+                "/buff",
+                "/jump 0-6",
+                "/run 0-7",
+                "/haste 0-3",                
+                "/invis",
+                "/vis"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 11 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: requires proof of strength

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+            'trail': '',
+            'op': 'false',
+            'speech': "Gre, greninja!",
+            'powers': array(
+                "/buff",
+                "/jump 0-6",
+                "/run 0-7",
+                "/haste 0-3",                
+                "/invis",
+                "/vis"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: requires proof of invisibility

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODc1OTkzMzg1NTAsInByb2ZpbGVJZCI6ImJkNmY2MjIyMGI3NTQ2OTg4YzgzZmExY2EyYWZiMmU5IiwicHJvZmlsZU5hbWUiOiJDaDFja2VuIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9iZjRhYmZmNmMzZGM5MDEyN2YxYWU3Yjc4MTJiZDdjNmRkMTk4ZTEzZjVjOGMzZDViNjQzZWY5ZTI4ZmEwYTAifX19',
+            'trail': '',
+            'op': 'false',
+            'speech': "Gre, greninja!",
+            'powers': array(
+                "/buff",
+                "/jump 0-6",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: jump should be limited to 4 unless you have proof he can jump really high

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+                "/invis",
+                "/vis"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 11 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 2 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 21 9999 19 true",
+                "/shot give ". @plr ." hydropump"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: this isnt a thing anymore

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 2 above)
Comment (ThisChicken) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: he's a water type frog, they have watersacs in them, they are amphibeous

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 5 above)
Comment (ThisChicken) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: he can learn camouflage in the games

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 6 above)
Comment (ThisChicken) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: http://static.fjcdn.com/gifs/Dcm_1f8681_5383610.gif

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 4 above)
Comment (ThisChicken) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: base attack stat of 95, and min at lvl.100 is 175

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 3 above)
Comment (ThisChicken) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: he can learn rest in the games. the move rest heals the user

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 5 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: please show me his camo ability

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 4 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: numbers mean nothing, show me the strongest thing he's ever done. video or pic only

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 3 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: requires pic or vid of regen

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 4 above)
Comment (ThisChicken) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: http://pa1.narvii.com/6062/5dc1586ddc9e2590f4c31edfd3607c322ab7f623_hq.gif that good nuf for ya?

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 5 above)
Comment (ThisChicken) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: i cannot find or capture video of greninja using this move, but i have a info site: http://pokemondb.net/pokedex/greninja
IK its not video or pics, but no one wants to upload a video of greninja using this move :/

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 3 above)
Comment (ThisChicken) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: can get pic of greninja using rest, but i have a list of moves that he can learn and rest is one of them.
no one likes this move, so i cant find it :/

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 5 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: denied, he appears to be teleporting

File: CommandHelper/LocalPackages/commands/buffs/greninja.ms (hunk 6 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/greninja.ms]: approved

**Pull request merged**